web/template/lpa.gohtml

```diff
@@ -3,164 +3,164 @@
+        <aside class="app-stat-block">
+          <h3 class="govuk-heading-l govuk-!-margin-bottom-2">{{ .Lpa.ObjectionCount }}</h3>
+          Objections
+        </aside>
+        <aside class="app-stat-block">
+          <h3 class="govuk-heading-l govuk-!-margin-bottom-2">{{ .Lpa.WarningCount }}</h3>
+          Warnings
+        </aside>
+        <aside class="app-stat-block">
+          <h3 class="govuk-heading-l govuk-!-margin-bottom-2">{{ .Lpa.ComplaintCount }}</h3>
+          Complaints
+        </aside>
+        <aside class="app-stat-block">
+          <h3 class="govuk-heading-l govuk-!-margin-bottom-2">{{ .Lpa.InvestigationCount }}</h3>
+          Investigations
+        </aside>
+        <aside class="app-stat-block">
+          <h3 class="govuk-heading-l govuk-!-margin-bottom-2">{{ len .TaskList }}</h3>
+          Tasks
+        </aside>
+      </div>
 
-    <h2>Tasks</h2>
+      <h2>Tasks</h2>
 
-    <ul data-role="tasks-list">
-      {{ range .TaskList }}
-        <li>{{ .Name }} ({{ .Assignee.DisplayName }})</li>
-      {{ end }}
-    </ul>
+      <ul data-role="tasks-list">
+        {{ range .TaskList }}
+          <li>{{ .Name }} ({{ .Assignee.DisplayName }})</li>
+        {{ end }}
+      </ul>
 
-    <h2 class="govuk-heading-m">Donor details</h2>
+      <h2 class="govuk-heading-m">Donor details</h2>
 
-    <dl class="govuk-summary-list">
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">Application format</dt>
-        <dd class="govuk-summary-list__value">
-          {{if (eq .Lpa.Application.Source "APPLICANT")}}
-            Digital
-          {{else}}
-            Paper
-          {{end}}
-        </dd>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">Name</dt>
-        <dd class="govuk-summary-list__value">{{ .Lpa.Application.DonorFirstNames }} {{ .Lpa.Application.DonorLastName }}</dd>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">Date of birth</dt>
-        <dd class="govuk-summary-list__value">{{ (date .Lpa.Application.DonorDob "2 January 2006") }}</dd>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">Address</dt>
-        <dd class="govuk-summary-list__value">
-          {{ .Lpa.Application.DonorAddress.Line1 }}
-          {{ if not (eq .Lpa.Application.DonorAddress.Line2 "") }}<br>{{ .Lpa.Application.DonorAddress.Line2 }}{{ end }}
-          {{ if not (eq .Lpa.Application.DonorAddress.Line3 "") }}<br>{{ .Lpa.Application.DonorAddress.Line3 }}{{ end }}
-        </dd>
-      </div>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">Town or city</dt>
-        <dd class="govuk-summary-list__value">{{ .Lpa.Application.DonorAddress.Town }}</dd>
-      </div>
-      {{ if not (eq .Lpa.Application.DonorAddress.Postcode "") }}
+      <dl class="govuk-summary-list">
```

Comment:
I havent changed any of this HTML, I only moved the two outer tags `<div class="govuk-grid-row">
  <div class="govuk-grid-column-full">` originally on line 6 & 7 down to line 17 & 18